src/element.ts

```diff
@@ -141,12 +141,26 @@ export abstract class Element {
    */
   setStyle(style: ElementStyle | undefined): this {
     this.style = style;
+    const svg = this.getSVGElement();
+    if (svg) {
+      if (style?.fillStyle) {
+        svg.setAttribute('fill', style.fillStyle);
```

Comment:
If the attribute is called `fill` then why don't we use the same naming scheme? I think calling it `fill` is simpler than `fillStyle`.

So the resulting line of code would look like:

```
svg.setAttribute('fill', style.fill);
```

Reply:
Ignore the above comment. I forgot that the SVG specification and the Canvas2D specification are different! I think in VexFlow, we are trying to adhere to the CanvasRenderingContext2D API as much as possible.

So here we are trying to support `CanvasRenderingContext2D.fillStyle`.